### PR TITLE
modify template folder logic for non default input

### PR DESCRIPTION
**Pull request by musm**
If users supplies own template folder, don't assume it is in the package directory

### CODE REVIEW

**Comment by musm** [comment]
cc @tpapp

File: skeleton.jl [hunk 1]
```diff
@@ -10,14 +10,13 @@ VERSION ≥ minver || err("need at least version $(minver)")
...
 isfile(destdir) && err("destination $(destdir) is a file")
-template = length(ARGS) == 2 ? ARGS[2] : "template"
+srcdir = length(ARGS) == 2 ? ARGS[2] : joinpath(@__DIR__, "template")
 pkgname = basename(destdir)
 
 if isempty(pkgname)
     pkgname = basename(dirname(destdir)) # add trailing /
 end
 
-
```
Comment (musm) [inline skeleton.jl:20]: there's a mysterious white space character here that sublime catched

**Comment by tpapp** [comment]
Sorry, somehow missed the original notification of the PR.

Thanks for catching this, I will merge.

**Comment by musm** [comment]
thanks

**Pull request merged**